elastic/eui
# [Docs] Update references to Sass/Emotion migration

## Summary

Code diffs can be a little hard to follow so I recommend following the QA steps below instead. I'm particularly looking for feedback on the developer experience and whether the copy/instructions would make sense to you as someone new to EUI (but not necessarily new to frontend dev).

## QA

- Go to https://eui.elastic.co/pr_8029/#/guidelines/getting-started
- [ ] Confirm the `Setting up your application` section no longer has any imported `.css` files
- [ ] Read through the `Styling your application` section and confirm that it makes sense with your understanding of EUI's usage and does not reference Sass.
- [ ] Also confirm that all links work properly/as expected in the two above sections
- Go to https://eui.elastic.co/pr_8029/#/theming/colors/values?themeLanguage=sass
- [ ] Confirm that a red danger callout appears at the top warning devs away from using our Sass utilities/values
- [ ] Switch the pink tab from Sass to CSS-in-JS and confirm the callout disappears
- Other docs/copy QA
- [ ] Spin up the `website`/EUI+ docs and confirm that the Getting Started page there matches `src-doc`'s
- [ ] Confirm the [repo README](https://github.com/elastic/eui/blob/50bed5678b3437e55ebd0a067973ca0fe4880900/README.md) no longer has a reference to the status of EUI's theme

### General checklist

N/A, docs only
